pubtools/_ami/tasks/push.py

```diff
@@ -448,6 +451,14 @@ def add_args(self):
             default=4,
         )
 
+        group.add_argument(
+            "--snapshot_accounts",
+            help="List of account ids to give snapshot creation permissions to if a new snapshot "
+                 "is created as part of the image push.",
+            type=list,
+            default=["679593333241", "684062674729", "425685993791", "582767206473"],
```

Comment:
Do you know whether these are meant to be kept as strings all the way through to the underlying API, or are they meant to be converted to integers somewhere?